### PR TITLE
feat: add videojs-airplay, videojs-chromecast

### PR DESCRIPTION
**Pull request by leonboripgs**
Hello, I created new PR for airplay/chromecast again regarding original one have some linting issue and structure problems.
Please review the PR and let me know your feedbacks.
Thanks.

### CODE REVIEW

File: src/components/templates/andPlaybackContext.tsx [hunk 1]
```diff
@@ -22,6 +22,8 @@ import {
 } from '@lib/generated/graphql';
 import hasVideo from '@lib/hasVideo';
 
+// import airplay from '../../lib/videojs-airplay/dist/silvermine-videojs-airplay';
```
Comment (narthur) [inline src/components/templates/andPlaybackContext.tsx:25]: Can we remove this?

File: src/components/templates/andPlaybackContext.tsx [hunk 2]
```diff
@@ -399,19 +409,33 @@ export default function AndPlaybackContext({
...
+						addButtonToControlBar: false,
+					},
+				},
 			};
 			if (playerRef.current) {
 				playerRef.current.src(sources);
 				resetPlayer();
 			} else if (videojs) {
 				const p = videojs.default(currentVideoEl, options);
+				console.log(p);
```
Comment (narthur) [inline src/components/templates/andPlaybackContext.tsx:425]: Maybe some testing code slipped through?

**Review by narthur — CHANGES_REQUESTED** [review]

File: src/components/templates/andPlaybackContext.tsx (hunk 1 above)
Comment (marcomiduri) [reply src/components/templates/andPlaybackContext.tsx]: @narthur I removed this code.

File: src/components/templates/andPlaybackContext.tsx (hunk 2 above)
Comment (marcomiduri) [reply src/components/templates/andPlaybackContext.tsx]: @narthur I removed this code.

**Review by narthur — APPROVED** [review]
Great work!

**Review by mattleff — CHANGES_REQUESTED** [review]
@leonboripgs Can you `nvm use` & `npm i` to ensure the lock file isn't changing versions?

**Comment by marcomiduri** [comment]
@mattleff To fix massive package-lock.json file update, I am closing this PR and created another cloned one. https://github.com/avorg/audioverse-next/pull/268

**Comment by marcomiduri** [comment]
@mattleff To fix massive package-lock.json file update, I am closing this PR and created another cloned one. https://github.com/avorg/audioverse-next/pull/268

**Pull request closed without merging**